src/components/BivariateLegend/const.ts

```diff
@@ -6,3 +8,30 @@ export const CORNER_POINTS_INDEXES = [
   BIVARIATE_LEGEND_SIZE * BIVARIATE_LEGEND_SIZE - BIVARIATE_LEGEND_SIZE,
   BIVARIATE_LEGEND_SIZE * BIVARIATE_LEGEND_SIZE - 1,
 ];
+
+export const LOW = `↓${i18n.t('bivariate.legend.low')}`;
+export const HIGH = `↑${i18n.t('bivariate.legend.high')}`;
+export const MEDIUM = i18n.t('bivariate.legend.medium');
+
+export const isTopSide = (index: number): boolean =>
+  Array.from(Array(BIVARIATE_LEGEND_SIZE), (_, i) => i).includes(index);
+
+export const isBottomSide = (index: number): boolean => {
+  const bottomRowStartIndex =
+    BIVARIATE_LEGEND_SIZE * BIVARIATE_LEGEND_SIZE - BIVARIATE_LEGEND_SIZE;
+  return Array.from(
+    Array(BIVARIATE_LEGEND_SIZE),
+    (_, i) => bottomRowStartIndex + i,
+  ).includes(index);
+};
+
+export const isLeftSide = (index: number): boolean =>
+  Array.from(Array(BIVARIATE_LEGEND_SIZE), (_, i) => i * BIVARIATE_LEGEND_SIZE).includes(
+    index,
+  );
+
+export const isRightSide = (index: number): boolean =>
+  Array.from(
+    Array(BIVARIATE_LEGEND_SIZE),
+    (_, i) => i * BIVARIATE_LEGEND_SIZE + (BIVARIATE_LEGEND_SIZE - 1),
+  ).includes(index);
```

Comment:
How about little explanation of those functions here? They look like helper functions but their not obvious

Reply:
added

Reply:
I'm curious why those arrays (`Array.from(Array(BIVARIATE_LEGEND_SIZE), (_, i) => i * BIVARIATE_LEGEND_SIZE)` etc) are not pre-computed?